orangecontrib/network/widgets/tests/test_OWNxGenerator.py

```diff
@@ -17,3 +17,17 @@ def test_balanced_tree(self):
         """
         balanced_tree = _balanced_tree(100)
         self.assertIsInstance(balanced_tree, Graph)
+
+    def test_regular_graph(self):
+        """
+        Random regular graph: inequality 0 <= d < n must be
+        satisfied where d is degree of each node and n is
+        the number of nodes.
+        GH-69
+        """
+        w = self.widget
+        w.controls.graph_type.setCurrentIndex(12)
+        self.assertEqual(GraphType.all[w.controls.graph_type.currentIndex()][0], "Regular")
```

Comment:
Could `setCurentIndex` to `GraphType.all.index('Regular')`?

Reply:
Ok, fixed.